### PR TITLE
Enabled INTEGRAL download

### PR DESCRIPTION
**Pull request by imbasimba**

### CODE REVIEW

File: astroquery/esasky/core.py [hunk 1]
```diff
@@ -1230,6 +1237,18 @@ def _get_maps_for_mission(self, maps_table, mission, download_dir, cache, json,
 
         return maps
 
+    def _ends_with_fits_like_extentsion(self, name):
```
Comment (bsipocz) [inline astroquery/esasky/core.py:1240]: Oh, what an arsenal of names for INTEGRAL files :)

**Review by bsipocz — APPROVED** [review]
All looks OK, and shall merge pending passing CI (and remote test).

**Comment by bsipocz** [comment]
Thanks @imbasimba!

File: astroquery/esasky/core.py (hunk 1 above)
Comment (imbasimba) [reply astroquery/esasky/core.py:1240]: Hehe, I imagine INTEGRAL is somewhat standardized regarding the extensions, but I decided to put all the versions I have encountered over the years, just to be safe.

**Pull request merged**